HonorAssist.lua

```diff
@@ -39,14 +39,18 @@ function HonorAssist:Initialize()
 end
 
 function HonorAssist:ProcessChatMsgCombatHonorGain(honorGainedSummary)
-	if string.match(honorGainedSummary, "dies, honorable kill") then
-		local estimatedHonorGained = string.match(honorGainedSummary, "%d+")
-		local playerKilled = string.match(honorGainedSummary, "^([^%s]+)")
-		local eventTimeUtc = HonorAssist:GetCurrentTimeUtc()
-		HonorAssist:AddKillToMasterDatabase(playerKilled, estimatedHonorGained, eventTimeUtc)
-		local honorGained = HonorAssist:AddKillToDailyDatabase(playerKilled, estimatedHonorGained, eventTimeUtc, HonorAssistLogging)
-		HonorAssist:AddKillToHourlyDatabase(honorGained, eventTimeUtc)
+	local estimatedHonorGained = string.match(honorGainedSummary, "%d+")
+	local playerKilled = string.match(honorGainedSummary, "^([^%s]+)")
+
+	-- It's a dishonorable kill. Just ignore it. 
+	if estimatedHonorGained == nil or playerKilled == nil then
```

Comment:
DK message has no number in it. Estimated Honor Gained will be nil and this will return immediately. It's why you had that entry in your saved variables w/ a DateUtc but no Honor.